go.mod

```diff
@@ -12,6 +12,7 @@ replace (
 // Direct and indirect dependencies are grouped together
 require (
 	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
+	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v0.3.0 // indirect; pinned
```

Comment:
Should this be moved below with other `indirect` deps?

Reply:
@pavannd1 That can be done.  I wanted to emphasize that the version needed to be pinned (just in case there was a change where `go mod tidy` decided that a newer version would work better).

Reply:
Please move this to the "indirect" block. IIUC, The convention in the Go ecosystem ATM, is to have 2 blocks, the first one with the direct dependencies, the second one with indirect dependencies. Go tools, such as `go get ...` will attempt to use and maintain that structure. At the same time, they do not move dependencies across blocks (AFAIK)

Include _"pinned"_ comment. It'd be good to include the reason for "pinning" it in the comment.

Reply:
@julio-lopez yes.  Thats my understanding - current Go convention is to have indirect after direct, generally in 2 blocks (although `tidy` may create one per tidy run).